portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,8 +35,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      String userInput = request.getParameter("user-input");
+      String userIdea = request.getParameter("user-input");
+      long timestamp = System.currentTimeMillis();
+
+      Entity youtubeEntity = new Entity("Youtube Idea");
+      youtubeEntity.setProperty("idea", userIdea);
+      youtubeEntity.setProperty("timestamp", timestamp);
+      
+      DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();      
```

Comment:
Lets make this a final instance variable of this class as we won't need a new instance of the datastore service for every post request